content/_help/fraud-concerns/fraud-concerts._en.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
There's a typo in the filename. The file `fraud-concerts._en.md` should be `fraud-concerns._en.md`.